tuning/autotune.py

```diff
@@ -9,18 +9,19 @@
 from pathlib import Path
 import time
 import multiprocessing
+import tune
 
 """
 Sample Usage:
 
-python autotune.py winograd 1286.mlir --Xtune='--lhs-dims=bmk' --Xtune='--rhs-dims=bkn' --Xtune='--tile-dims=*mnk' --devices=1,3,5 --num-candidates=64
+python autotune.py winograd 1286.mlir --lhs-dims=bmk --rhs-dims=bkn --tile-dims=*mnk --devices=1,3,5 --num-candidates=64
 
 """
 
 
 # Default values for num_candidates and devices, change it as needed
-DEFAULT_NUM_CANDIDATES = 1024
-DEFAULT_DEVICE_LIST = [0]
+DEFAULT_NUM_CANDIDATES = 2048
+DEFAULT_DEVICE_LIST = [1, 3]
```

Comment:
could you change it back to 0? This seems like a safer default for systems with only one gpu